spec/index.html

```diff
@@ -2360,16 +2318,19 @@ <h2>Changes between RDF 1.1 and RDF 1.2</h2>
       for representing <a data-cite="RDF12-CONCEPTS#dfn-triple-term">triple terms</a> in Turtle.</li>
     <li>Changes the `LANGTAG` terminal production to
       <a href="#grammar-production-LANG_DIR" class="type langDir"><code>LANG_DIR</code></a> to include
-      an optional <a data-cite="RDF12-CONCEPTS#dfn-base-direction">base direction</a>.</li>
+      an optional <a data-cite="RDF12-CONCEPTS#dfn-base-direction">initial text direction</a>.</li>
     <li>Clarify that 
       <a data-cite="I18N-GLOSSARY#dfn-surrogate" class="lint-ignore">Unicode surrogates</a> 
       are not legal in Turtle documents, nor as the value of a Unicode escape sequence.</li>
     <li>Added <a href="#sec-version"></a> and parser
-      state and productions to announce the RDF version associated with the input document.</li>
+      state and productions to announce the RDF version
+      associated with the input document.
+      Note that this also includes a provision that `@version` does not match
+      <a href="#grammar-production-LANG_DIR" class="type langDir"><code>LANG_DIR</code></a>;
+      the previous version of the grammar would allow such a match,
+      but would result in a literal with a non-well-formed language tag accroding to [[BCP47]], and form an invalid literal.</li>
```

Comment:
`non-well-formed` feels clumsy. BCP47 (consisting of [RFC 4647](https://www.rfc-editor.org/info/rfc4647) and [RFC 5646](https://www.rfc-editor.org/info/rfc5646)) does not contain `non-well-formed` anywhere, but it *does* have one instance of `ill-formed` (and 17 instances of `well-formed). Can we use `ill-formed` instead of `non-well-formed`?
```suggestion
      but would result in a literal with an ill-formed language tag according to [[BCP47]], and form an invalid literal.</li>
```